### ISSUE TITLE
Thoughts on Relay?

### ISSUE THREAD

**Issue by necrodome** [andreypopp/react-async#40]
Hi, do you have any thoughts on [Facebook's Relay](https://gist.github.com/wincent/598fa75e22bdfa44cf47)?
It turns out react-async (+ the router) is more on par for a better model than flux implementations.

**Comment by mhuebert** [comment]
I agree. I was using react-async last year for exactly this same purpose. react-async provided a great way to declare data dependencies inside each component.

**Comment by RickWong** [comment]
Hey guys, check out https://github.com/RickWong/react-transmit (Relay-ish) I'm planning to extend it with `react-async`.
